sky/provision/nebius/instance.py

```diff
@@ -0,0 +1,260 @@
+                    head_instance_id = stopped_instance_id
+            except Exception as e:  # pylint: disable=broad-except
+                logger.warning(f'Start instance error: {e}')
+                raise
+            sleep(utils.POLL_INTERVAL)  # to avoid fake STOPPED status
+            logger.info(f'Started instance {stopped_instance_id}.')
+
+    for _ in range(to_start_count):
+        node_type = 'head' if head_instance_id is None else 'worker'
+        try:
+            platform, preset = config.node_config['InstanceType'].split('_')
+            instance_id = utils.launch(
+                cluster_name_on_cloud=cluster_name_on_cloud,
+                instance_type=node_type,
+                platform=platform,
+                preset=preset,
+                region=region,
+                image_family=config.node_config['ImageId'],
+                disk_size=config.node_config['DiskSize'],
+                user_data=config.node_config['UserData'])
+        except Exception as e:  # pylint: disable=broad-except
+            logger.warning(f'run_instances error: {e}')
+            raise
+        logger.info(f'Launched instance {instance_id}.')
+        created_instance_ids.append(instance_id)
+        if head_instance_id is None:
+            head_instance_id = instance_id
+    assert head_instance_id is not None, 'head_instance_id should not be None'
+    return common.ProvisionRecord(provider_name='nebius',
+                                  cluster_name=cluster_name_on_cloud,
+                                  region=region,
+                                  zone=None,
+                                  head_instance_id=head_instance_id,
+                                  resumed_instance_ids=resumed_instance_ids,
+                                  created_instance_ids=created_instance_ids)
+
+
+def wait_instances(region: str, cluster_name_on_cloud: str,
+                   state: Optional[status_lib.ClusterStatus]) -> None:
+    del state
+    _wait_until_no_pending(region, cluster_name_on_cloud)
+
+
+def stop_instances(
+    cluster_name_on_cloud: str,
+    provider_config: Optional[Dict[str, Any]] = None,
+    worker_only: bool = False,
+) -> None:
+    if provider_config is not None and provider_config['region']:
+        exist_instances = _filter_instances(provider_config['region'],
+                                            cluster_name_on_cloud, ['RUNNING'])
+        for instance in exist_instances:
+            if worker_only and instance.endswith('-head'):
+                continue
+            utils.stop(instance)
+
+
+def terminate_instances(
+    cluster_name_on_cloud: str,
+    provider_config: Optional[Dict[str, Any]] = None,
+    worker_only: bool = False,
+) -> None:
+    """See sky/provision/__init__.py"""
+    if provider_config is not None and provider_config['region']:
+        instances = _filter_instances(provider_config['region'],
+                                      cluster_name_on_cloud, None)
```

Comment:
```suggestion
                                      cluster_name_on_cloud, status_filters=None)
```
nit: use kwargs when argument is literal value. same for other appearances in this file